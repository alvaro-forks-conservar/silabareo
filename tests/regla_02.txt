bajá
cadavérico
salí
sacó
palúdico
errático
dejé
pelícano
esófago
betún
limáis
icéis
isógono
tisú
ahorráis
coméis
podólogo
obús
usáis
emulé
rubí
turrón
zulú
milímetro
inverosímil