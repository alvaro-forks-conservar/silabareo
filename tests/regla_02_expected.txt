ba-já
ca-da-vé-ri-co
sa-lí
sa-có
pa-lú-di-co
e-rrá-ti-co
de-jé
pe-lí-ca-no
e-só-fa-go
be-tún
li-máis
i-céis
i-só-go-no
ti-sú
a-ho-rráis
co-méis
po-dó-lo-go
o-bús
u-sáis
e-mu-lé
ru-bí
tu-rrón
zu-lú
mi-lí-me-tro
in-ve-ro-sí-mil